src/services/withdrawal.py

```diff
@@ -11,6 +11,13 @@
 
 
 class Withdrawal:
+    """
+    Each oracle report finalizes a certain number of withdrawal requests. To determine how many requests will be
```

Comment:
Service calculates which withdrawal requests should be finalized using next factors:
- ...